install_config/configuring_authentication.adoc

```diff
@@ -128,81 +128,132 @@ challenges should set this header to a non-empty value.
 Not supported by all identity providers.
 
 If you want users to be sent to a branded page before being redirected to
-the identity provider's login, then set `*oauthConfig -> alwaysShowProviderSelection: true*`
-in the master configuration file. This provider selection page can be
+the identity provider's login, then set `*oauthConfig -> alwaysShowProviderSelection: true*`. This provider selection page can be
 xref:../install_config/web_console_customization.adoc#customizing-the-login-page[customized].
 
 |`mappingMethod`  | Defines how new identities are mapped to users when they login. See xref:mapping-identities-to-users[Mapping Identities to Users] for more information.
 |===
 
-[NOTE]
 When adding or changing identity providers, you can map identities from the new
 provider to existing users by setting the `*mappingMethod*` parameter to
 `*add*`.
 
+After saving changes to the master configuration file, you must restart the {product-title} services for the changes to take effect:
+
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+----
+endif::[]
+
+
 [[mapping-identities-to-users]]
 
-=== Mapping Identities to Users
+== Mapping Identities to Users
 
 Setting the `*mappingMethod*` parameter in a
 xref:../install_config/master_node_configuration.adoc#install-config-master-node-configuration[master configuration file]
-determines how identities are mapped to users:
+determines how identities are mapped to users.
 
+To map identities to users:
+
+. Edit the master configuration file:
++
+----
+/etc/origin/master/master-config.yaml
+----
++
+.Master Configuration for Mapping Identities
 ====
+
 ----
 ...
 oauthConfig:
   identityProviders:
   - name: htpasswd_auth
     challenge: true
-    login: false
-    mappingMethod: "claim"
+    login: false <1>
```

Comment:
remove this footnote number, only the `mappingMethod` is important in this section